docs/blog/2019-08-30-building-the-net-third-and-grove-website-in-gatsby/index.md

```diff
@@ -0,0 +1,65 @@
+
+## What we love about TAGV5
+
+We love how our new brand is portrayed on the site but we’ll focus on the benefits of the technologies we used.
+
+**Performance.** One of the great Gatsby claims is performance. Out of the gate their starter has incredible scores in chrome lighthouse. Our worst performance metric is in image load times, like many other sites across the web. Manually going through and processing images, or even doing automated batch processes is a pain and can sometimes become a rabbit hole. Their ‘gatsby-image’ plugin does some awesome pre-processing, resizing, and even lazy loading so you can maintain these great load times. So maintaining that high-performance score is not a chore, but a joy.
+
+**Preview ❤️. **Our site is the first in the world to use [Drupal with Gatsby Preview](https://www.gatsbyjs.org/blog/2019-06-26-live-preview-for-drupal/) in production (mainly because we built Drupal + Gatsby preview). What this allows content editors to see is an actual rendering of the content they are working on. Not an approximation, a fully rendered React app with their content as they are editing it. It's in the early stages but Third and Grove developed both the Drupal module and Gatsby source plugin modifications to make this possible for anyone to do.
+
+We feel like this allows content editors to keep using tools they are comfortable with. It also opens up opportunities to implement workflows that were harder to set up previously. One might be having the Gatsby Preview instance as a staging environment, so content is previewed there and approved before pushing to a live site. We’re excited to explore different options this allows and hope to get community feedback on how others are using this great new tooling.
+
+## Pain points that we came across (and solved)
+
+We’re going to get technical here because the main pain points were around custom modifications to get exactly what we wanted.
+
+**Styling. **One of the more contentious issues in the JavaScript community is styling. CSS in JS, just that term, stirs up some mighty opinions. This won’t address the rift, we are firmly on the side of ‘css in js is dope’. Using Emotion in our Gatsby project has been amazing. It has allowed us to maintain a very clean project structure, without sacrificing any of the flexibility of using SASS. We could have easily used SASS in this project as Gatsby has support for it. Instead of having matching co-located style files, we write local styles directly in the component where they are scoped. If you're unfamiliar with scoping styles, it allows you to write styles with no fear of them leaking to an unintended target somewhere else in the DOM.
```

Comment:
```suggestion
**Styling.** One of the more contentious issues in the JavaScript community is styling. CSS in JS, just that term, stirs up some mighty opinions. While this won’t address the rift, we are firmly on the side of ‘css in js is dope’. Using Emotion in our Gatsby project has been amazing. It has allowed us to maintain a very clean project structure, without sacrificing any of the flexibility of using Sass. We could have easily used Sass in this project as Gatsby has support for it. Instead of having matching co-located style files, we write local styles directly in the component where they are scoped. If you're unfamiliar with scoping styles, it allows you to write styles with no fear of them leaking to an unintended target somewhere else in the DOM.
```